tuf_conformance/test_file_download.py

```diff
@@ -135,10 +140,9 @@ def test_multiple_changes_to_target(
         new_file_contents = f"new file contents {i}".encode()
         new_target_path = f"new-target-{i}"
         repo.add_artifact(Targets.type, new_file_contents, new_target_path)
-        repo.targets.version += 1
 
         # Bump repo snapshot
```

Comment:
Could specify that we also bump targets and timestamp.

Reply:
https://github.com/theupdateframework/tuf-conformance/pull/188/commits/abacf0b7b4bd7d2ba7f412802f3d3f10a2718f74